README.md

```diff
@@ -5,3 +5,39 @@ This is the Git repo of the Docker [official image](https://docs.docker.com/dock
 ## Release Process
 
 The release process is documented on our [Wiki Page](https://github.com/ExpressGateway/express-gateway/wiki/Express-Gateway-Release-Process)
+
+## Using this docker image
+
+Obtain a local copy of the image:
+```
+docker pull expressgateway/express-gateway:latest
```

Comment:
This is not correct. Once published the image would be just `express-gateway`

Reply:
Hi XVincentX.  I think you are suggesting to remove the ":latest" tag from the image name on the docker pull instructions.  That is fine, it defaults to "latest" if no tag is provided. I tend to prefer explicit tags which is why I added it.  I will make that change and resubmit.  Thanks.

Reply:
Hmm no, that was not my intention.

Let me clarify: the current Docker Image we're publishing on [DockerHub](https://hub.docker.com/r/expressgateway/express-gateway/) is not generated using this file, but [another one](https://github.com/ExpressGateway/express-gateway/blob/master/Dockerfile).

This repository is to incorporate Express Gateway as an [official Docker Image](https://github.com/docker-library/official-images/pull/3980) — which currently is still waiting to be merged. When that happens, the official image name will simply be `express-gateway`.